pkg/app/destroy.go

```diff
@@ -96,5 +98,12 @@ func destroy(clicontext *cli.Context) error {
 		logrus.Infof("failed to remove entry %s from your SSH config file: %s", ctrName, err)
 		return errors.Wrap(err, "failed to remove entry from your SSH config file")
 	}
+
+	homeDirectory := syncthing.GetHomeDirectory(name)
+	err = os.RemoveAll(homeDirectory)
+	if err != nil {
+		logrus.Error(err)
+	}
```

Comment:
Is it related to CleanLocalConfig?

Reply:
clean local config is based on a syncthing struct, which is not present in the destroy command (technically i can construct it and then use CleanLocalConfig). Or just write another function that concise all of this.